src/Take.Blip.Builder/Actions/ProcessHttp/ProcessHttpAction.cs

```diff
@@ -89,6 +96,16 @@ public override async Task ExecuteAsync(IContext context, ProcessHttpSettings se
 
         }
 
+        private void AssertDenyIps(Uri uri)
+        {
+            Regex re = new Regex($"{_configuration.IpsDeniedOnHttpAction}");
```

Comment:
what happen if "IpsDeniedOnHttpAction" has an empty value?
I belive that be better do a "return" to avoid create a new regex instance...

Reply:
Done!